pyaedt/modules/PostProcessor.py

```diff
@@ -945,6 +945,59 @@ def steal_focus_oneditor(self):
         self.oeditor.Delete(["NAME:Selections", "Selections:=", "DUMMYSPHERE1"])
         return True
 
+    @aedt_exception_handler
+    def export_report_to_csv(self, project_dir, plot_name):
+        """Export the SParameter plot data to a CSV file.
+
+        This method leaves the data in the plot (as data) as a reference
+        for the Sparameters plot after the loops.
+
+        Parameters
+        ----------
+        project_dir : str
+            Path to the project directory.
+        plot_name : str
+            Name of the plot to export.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.)
+        """
+        # path
+        npath = os.path.normpath(project_dir)
+        # set name for the csv file
+
+        csvFileName = os.path.join(npath, plot_name + ".csv")
+        # export the csv
+        self.oreportsetup.ExportToFile(plot_name, csvFileName)
```

Comment:
`ExportToFile` is already self-documenting.
```suggestion
        csv_file_name = os.path.join(npath, plot_name + ".csv")
        self.oreportsetup.ExportToFile(plot_name, csv_file_name)
```